jparest/src/main/java/uk/gov/homeoffice/digital/sas/jparest/config/ObjectMapperConfig.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.homeoffice.digital.sas.jparest.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.hibernate5.Hibernate5Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ObjectMapperConfig {
+
+    @Bean
+    public ObjectMapper objectMapper() {
```

Comment:
this code could be moved to ApplicationConfig

Reply:
`ObjectMapperConfig` is imported in `ApplicationConfig`. As explained in my response to Stephen's comment above, the Object Mapper config was extracted in its own file to allow flexibility and re-use in tests